gulpfile.js

```diff
@@ -84,6 +84,7 @@ function buildExtensions(options) {
   buildExtension('amp-fit-text', '0.1', true, options);
   buildExtension('amp-font', '0.1', false, options);
   buildExtension('amp-iframe', '0.1', false, options);
+  buildExtension('amp-ima', '0.1', false, options);
```

Comment:
@shawnbuso this needs to be `amp-ima-player`. Without that AMP is building the wrong thing and it'll not load the extension correctly.

After fixing that I am seeing these errors:
```
videojs.ads.js:719 Uncaught ReferenceError: videojs is not defined(anonymous function) @ videojs.ads.js:719
videojs.ima.js:1191 Uncaught ReferenceError: videojs is not defined(anonymous function) @ videojs.ima.js:1191(anonymous function) @ videojs.ima.js:1192
ima.js:67 Uncaught TypeError: player.ima is not a function(anonymous function) @ ima.js:67
```